cypress/e2e/pages/load_safe.pages.js

```diff
@@ -1,13 +1,18 @@
 import * as constants from '../../support/constants'
 import * as sidebar from '../pages/sidebar.pages'
 import * as main from '../pages/main.page'
+import { addressBookRecipient } from '../pages/address_book.page'
 
 const addExistingAccountBtnStr = 'Add existing one'
 const contactStr = 'Choose address, network and a name'
 export const invalidAddressFormatErrorMsg = 'Invalid address format'
 const invalidAddressNameLengthErrorMsg = 'Maximum 50 symbols'
 
 const safeDataForm = '[data-testid=load-safe-form]'
+const removeOwnerBtn = '[data-testid="remove-owner-btn"]'
```

Comment:
const removeOwnerBtn = '[data-testid="remove-owner-btn"]'
const addOwnerBtn = '[data-testid="add-owner-btn"]' 
not sure that load_safe.page is the right place for these consts, as in general it's not possible to add or remove owner for existing safe during loading process

Reply:
what do you think about splitting into create_safe.pages and load_safe.pages ?

Reply:
I suggest we do these and other tasks in scope of refactoring tasks later. Otherwise we will spend much time on just that.

Reply:
Ok, let's add a task to our automation board and do refactoring later